connectors/spark/src/main/scala/io/unitycatalog/connectors/spark/AbfsVendedTokenProvider.java

```diff
@@ -0,0 +1,25 @@
+package io.unitycatalog.connectors.spark;
+
+import org.apache.hadoop.fs.azurebfs.extensions.SASTokenProvider;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.security.AccessControlException;
+
+import java.io.IOException;
+
+import static java.lang.String.format;
+
+public class AbfsVendedTokenProvider implements SASTokenProvider {
```

Comment:
not related to this PR, do we have document to tell uses how to set this provider in Spark configs?

Reply:
Not yet -- @dennyglee and I have been working on docs for this.  I'm planning on getting that polished up once we get changes in and confirm everything is working as expected